po/fr.po

```diff
@@ -31198,152 +31251,136 @@ msgstr ""
-#, fuzzy
 msgid "Print # of in packets"
-msgstr "Afficher tous les noms"
+msgstr "Afficher le nombre de paquets en entrée"
 
 #: /tmp/fish/implicit/share/completions/ifdata.fish:13
-#, fuzzy
 msgid "Print # of in bytes"
-msgstr "Afficher les fiches complètes"
+msgstr "Afficher le nombre d’octets en entrée"
 
 #: /tmp/fish/implicit/share/completions/ifdata.fish:14
-#, fuzzy
 msgid "Print # of in errors"
-msgstr "Afficher les fiches complètes"
+msgstr "Afficher le nombre d’erreurs en entrée"
 
 #: /tmp/fish/implicit/share/completions/ifdata.fish:15
-#, fuzzy
 msgid "Print # of in drops"
-msgstr "Afficher les informations APM"
+msgstr "Afficher le nombre de paquets ignorés en entrée"
 
 #: /tmp/fish/implicit/share/completions/ifdata.fish:16
 msgid "Print # of in fifo overruns"
-msgstr ""
+msgstr "Afficher le nombre de dépassements de file en entrée"
 
 #: /tmp/fish/implicit/share/completions/ifdata.fish:17
-#, fuzzy
 msgid "Print # of in compress"
-msgstr "Afficher les ratios de compression"
+msgstr ""
 
 #: /tmp/fish/implicit/share/completions/ifdata.fish:18
 msgid "Print # of in multicast"
-msgstr ""
+msgstr "Afficher le nombre de paquets multidiffusés en entrée"
 
 #: /tmp/fish/implicit/share/completions/ifdata.fish:19
 msgid "Print all statistics on output"
-msgstr ""
+msgstr "Afficher toutes les statistiques des sorties"
 
 #: /tmp/fish/implicit/share/completions/ifdata.fish:20
-#, fuzzy
 msgid "Print # of out packets"
-msgstr "Version du paquet source"
+msgstr "Afficher le nombre de paquets en sortie"
 
 #: /tmp/fish/implicit/share/completions/ifdata.fish:21
-#, fuzzy
 msgid "Print # of out bytes"
-msgstr "Envoyer les fichiers vers le tube stdout"
+msgstr "Afficher le nombre d’octets en sortie"
 
 #: /tmp/fish/implicit/share/completions/ifdata.fish:22
-#, fuzzy
 msgid "Print # of out errors"
-msgstr "Afficher les fiches complètes"
+msgstr "Afficher le nombre d’erreurs en sortie"
 
 #: /tmp/fish/implicit/share/completions/ifdata.fish:23
-#, fuzzy
 msgid "Print # of out drops"
-msgstr "Afficher les fiches complètes"
+msgstr "Afficher le nombre de paquets ignorés en entrée"
 
 #: /tmp/fish/implicit/share/completions/ifdata.fish:24
 msgid "Print # of out fifo overruns"
-msgstr ""
+msgstr "Afficher le nombre de dépassements de file en sortie"
 
 #: /tmp/fish/implicit/share/completions/ifdata.fish:25
-#, fuzzy
 msgid "Print # of out collisions"
-msgstr "Afficher toutes les versions"
+msgstr "Afficher le nombre de collisions de paquets en sortie"
 
 #: /tmp/fish/implicit/share/completions/ifdata.fish:26
 msgid "Print # of out carrier loss"
-msgstr ""
+msgstr "Afficher le nombre de pertes de porteuse en sortie"
 
 #: /tmp/fish/implicit/share/completions/ifdata.fish:27
 msgid "Print # of out multicast"
-msgstr ""
+msgstr "Afficher le nombre de paquets multidiffusés en entrée"
 
 #: /tmp/fish/implicit/share/completions/ifdata.fish:28
 msgid "Print # of incoming bytes per second"
-msgstr ""
+msgstr "Afficher le débit en entrée en o.s⁻¹"
```

Comment:
That seems overly technical (same for next message).

Reply:
At some point, I set myself the arbitrary goal to try to make my translations have a shorter character count than the untranslated text, as french was often making them longer than the original text. A combination of fatigue, ego and subconscious jingoism, I assume.

Reply:
Same as my remark about octal notation: the terseness is only a win if it is not gained at the expense of the ability of users to understand the message, and decrypting _octets par seconde_ in exponential notation does not seem to fit that bill.